src/app/files.js

```diff
@@ -0,0 +1,98 @@
+'use strict'
+
+var EventManager = require('../lib/eventManager')
+
+function Files (storage) {
+  var event = new EventManager()
+  this.event = event
+  var readonly = {}
+
+  this.exists = function (path) {
+    // NOTE: ignore the config file
+    if (path == '.browser-solidity.json') {
```

Comment:
This is kind of a "special file" or system file, so perhaps we can hide it only at the UI level? Although that might be more difficult.